docs/DeveloperGuide.adoc

```diff
@@ -277,90 +277,240 @@ Refer to the guide <<DevOps#, here>>.
+|`* * *` |user |view all notes related to a group |
+|`* *` |user who works with the same person in multiple groups |view all notes related to these groups as well when querying for the person |
+|`* *` |user with many friends with the same name |be able to differentiate them easily |locate a specific person
+|`* *` |user |archive old notes |keep them while not cluttering my app page
+|`* *` |clumsy user |export all my data and create a backup |keep my data somewhere safe
+|`* *` |forgetful user |get suggestions when typing commands |do not need to memorize commands
+|`*` |user |embed links in my notes |directly access relevant webpages
+|`*` |user |associate photos with notes |organize my photos better
+|`*` |user |add text styles |personalize my entries
+|`*` |user |colour entries with the same tag |organize my notes better
+|`*` |clumsy user |prompted for correction when I make typos |fix my command without re-typing it entirely
+|`*` |clumsy user |prompted for confirmation when I delete or edit notes or contacts |avoid making irreversible mistakes
+|`*` |user |lock notes with authentication |keep my notes secure
+|=======================================================================
 
-|`* * *` |user |delete a person |remove entries that I no longer need
+[appendix]
+== Use Cases
 
-|`* * *` |user |find a person by name |locate details of persons without having to go through the entire list
+(For each of the use cases below, the *System* is `TagLine` and the *Actor* is the `user`, unless specified otherwise)
 
-|`* *` |user |hide <<private-contact-detail,private contact details>> by default |minimize chance of someone else seeing them by accident
+[discrete]
+=== UCC01 Add person
 
-|`*` |user with many persons in the address book |sort persons by name |locate a person easily
-|=======================================================================
+*MSS*
 
-_{More to be added}_
+1.  User requests to add a contact.
+2.  TagLine adds the contact to the contact list.
++
+Use case ends.
 
-[appendix]
-== Use Cases
+*Extensions*
 
-(For all use cases below, the *System* is the `AddressBook` and the *Actor* is the `user`, unless specified otherwise)
+[none]
+* 1a. [underline]#UCE01 Invalid command syntax#
++
+[none]
+* 1b. [underline]#UCE02 Missing compulsory fields#
 
 [discrete]
-=== Use case: Delete person
+=== UCC02 Add person
 
 *MSS*
 
-1.  User requests to list persons
-2.  AddressBook shows a list of persons
-3.  User requests to delete a specific person in the list
-4.  AddressBook deletes the person
+1.  User requests to create a new group.
+2.  TagLine creates the group.
 +
 Use case ends.
 
 *Extensions*
 
 [none]
-* 2a. The list is empty.
+* 1a. [underline]#UCE01 Invalid command syntax#
++
+[none]
+* 1b. [underline]#UCE02 Command with missing compulsory fields#
++
+[none]
+* 1c. User inputs a group that already exists.
++
+[none]
+** 1c1. TagLine adds the alternate name to the group.
```

Comment:
Does `alternate name` mean the group is assigned an alias, or does it mean that the user enters a different name other than the existing one?